src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public interface Figure {
+
+    public abstract double getArea();
+    public abstract void draw();

```

Comment:
These 2 methods should be in separate interfaces
And class figure should implement them